Team6220_PowerPlay/src/main/java/org/firstinspires/ftc/team6220_PowerPlay/testclasses/ColorDetectPipeline.java

```diff
@@ -0,0 +1,112 @@
+package org.firstinspires.ftc.team6220_PowerPlay.testclasses;
+
+import org.opencv.core.Core;
+import org.opencv.core.Mat;
+import org.opencv.core.MatOfPoint;
+import org.opencv.core.Rect;
+import org.opencv.core.Scalar;
+import org.opencv.core.Size;
+import org.opencv.imgproc.Imgproc;
+import org.openftc.easyopencv.OpenCvPipeline;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ColorDetectPipeline extends OpenCvPipeline {
+    private Size blurSize = new Size(25, 25);
+    private double erodeSize = 220;
+
+    public boolean isRunning = false;
+    public Rect detectedRect = new Rect();
+    public boolean rectDetected = false;
```

Comment:
These are some potentially confusing variable names

Reply:
the first one is the actual rectangle bounding box around the detected object and the second one is telling you if there is a bounding box around the object of not

Reply:
Yeah that is clear from the types, but it should also be clear from the variable names since the person using them may not go back to the declaration to check the type. Please update the variables names.

Reply:
File no longer exists